### PR TITLE
allow none configtype for standalone testing other zookeeper clients

### PR DESCRIPTION
**Pull request by aisch**
wanted to use this w/ local testing (so w/o s3) of https://github.com/jaytaylor/ansible-kafka

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,22 +3,27 @@ language: python
...
   - pip install ansible==$ANSIBLE_VERSION
-  - "cp tests/ansible.cfg ansible.cfg"
-  - ansible-galaxy install -r tests/requirements.yml
+  - cd tests
+  - ansible-galaxy install -r requirements.yml
 
 script:
-  - "ansible-playbook -i tests/inventory tests/playbook.yml --syntax-check"
-  - "ansible-playbook -i tests/inventory tests/playbook.yml --connection=local --sudo"
+  - "ansible-playbook -i 'localhost,' playbook.yml --extra-vars=$ANSIBLE_EXTRA_VARS --syntax-check"
```
Comment (aisch) [inline .travis.yml]: looks like ansible has this secret 1-off inventory spec: `'localhost,'`

File: tests/ansible.cfg [hunk 2]
```diff
@@ -1,2 +1,2 @@
 [defaults]
-roles_path = ../
+roles_path = ../../
```
Comment (aisch) [inline tests/ansible.cfg]: we'll just run tests from `./tests`

**Comment by jacobwgillespie** [comment]
Is this ready to merge?

**Comment by aisch** [comment]
@jacobwgillespie i think so. all changes are additive (i.e. leave default behavior the same) and we've added a travis build matrix to test both config types (`s3` and `none`).

**Pull request merged**